Octokit/GitHubClient.cs

```diff
@@ -27,6 +27,23 @@ public GitHubClient(ProductHeaderValue productInformation)
         {
         }
 
+
+        /// <summary>
+        /// Create a new instance of the GitHub API v3 client pointing to 
+        /// https://api.github.com/
+        /// </summary>
+        /// <param name="productInformation">
+        /// The name (and optionally version) of the product using this library. This is sent to the server as part of
+        /// the user agent for analytics purposes.
+        /// </param>
+        /// <param name="httpTimeout">
+        /// The number of milliseconds to wait before the request times out. The default HTTP timeout is 100000.
```

Comment:
This comment isn't accurate. The parameter is a `TimeSpan`, not an integer representing milliseconds.